test/libsolidity/syntaxTests/immutable/unary.sol

```diff
@@ -0,0 +1,9 @@
+contract A {
+  int immutable a;
+  constructor() { a = 5; }
+  function f() public { --a; }
+}
+
+// ----
+// TypeError 1581: (85-86): Immutable variables can only be initialized inline or assigned directly in the constructor.
+// TypeError 1574: (85-86): Immutable state variable already initialized.
```

Comment:
This is error is not really bad, but kinda misleading.

Reply:
I will check if I can improve it. NB: it should complain about the writing, not about the re-writing.